### DISCUSSION TITLE
StateTimeline: Show value mappings in legend

### DISCUSSION THREAD

**Discussion by pepjo** [grafana/grafana#38315] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
In the new state map added in Grafana 8, the information displayed in the legend comes from the thresholds' configuration. However, no name can be assigned to each region in that configuration. Instead, it would be great if the value mappings can be shown.

This is my current configuration:
![image](https://user-images.githubusercontent.com/1307305/123618452-2064ab00-d800-11eb-9cf8-09482dac8f91.png)

I would suggest doing something like the flant status map Grafana plugin:
![image](https://user-images.githubusercontent.com/1307305/123618647-50ac4980-d800-11eb-9fe5-ea91963860a4.png)

**Why is this needed**:
To have a legend that is meaningful.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by wdlaszlo** [comment]
Something else that is also useful is to have the possibility to show the duration in the legend, something similar with what [Discrete Panel](https://grafana.com/grafana/plugins/natel-discrete-panel/) provides.

![Screenshot from 2022-07-26 12-32-23](https://user-images.githubusercontent.com/30290906/180974466-c4a0ef2a-2598-4ad7-8eac-25f9ba9f8f19.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
